src/importers/onepassword1PifImporter.ts

```diff
@@ -167,6 +171,36 @@ export class OnePassword1PifImporter extends BaseImporter implements Importer {
                     // Skip since brand was determined from number above
                     return;
                 }
+            } else if (cipher.type === CipherType.Identity) {
+                const identity = cipher.identity;
+
+                if (this.isNullOrWhitespace(identity.firstName) && fieldDesignation === 'firstname') {
+                    identity.firstName = fieldValue;
+                    return;
+                } else if (this.isNullOrWhitespace(identity.lastName) && fieldDesignation === 'lastname') {
+                    identity.lastName = fieldValue;
+                    return;
+                } else if (this.isNullOrWhitespace(identity.middleName) && fieldDesignation === 'initial') {
+                    identity.middleName = fieldValue;
+                    return;
+                } else if (this.isNullOrWhitespace(identity.phone) && fieldDesignation === 'defphone') {
+                    identity.phone = fieldValue;
+                    return;
+                } else if (this.isNullOrWhitespace(identity.company) && fieldDesignation === 'company') {
+                    identity.company = fieldValue;
+                    return;
+                } else if (this.isNullOrWhitespace(identity.email) && fieldDesignation === 'email') {
+                    identity.email = fieldValue;
+                    return;
+                } else if (fieldDesignation === 'address') {
+                    // fieldValue is an object casted into a string, so access the plain value instead
+                    const { street, city, country, zip } = field[valueKey];
+                    identity.address1 = street;
```

Comment:
You said these are exported as blank strings if not provided. Should we perform `isNullOrWhitespace` checks on each here? ex: `identity.city = this.isNullOrWhitespace(city) ? null : city`. I'd hate to store an encrypted string of an empty field if that is all it is.

Reply:
Sure. I assumed that's handled sometime later when preparing the payload as I couldn't "see" these properties in my vault's test record. I'll change it.

Reply:
It might be, can’t remember.  but we can just handle it here to be sure.